Makefile

```diff
@@ -1,30 +1,30 @@
 .PHONY: build release install uninstall clean test doc reindent
 
 build:
-	jbuilder build @install --dev
+	dune build @install
 
 release:
-	jbuilder build @install
+	dune build --profile release @install
```

Comment:
You'll need to update the specfile too to use `make release` instead of calling jbuilder directly, probably best to update it to use the Makefile rules so we do not need to keep them in sync.